### PR TITLE
Add config to try and build the websocket only image

### PR DESCRIPTION
**Pull request by jon-betts**
This doesn't try and deploy or run it anywhere yet.

This requires: https://github.com/hypothesis/pipeline-library/pull/6 before it can work.

This is to try out building multiple images from the same repo.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -47,15 +49,16 @@ node {
 
     onlyOnMaster {
         stage('release') {
-            releaseApp(image: img)
+            releaseApp(image: image)
+            releaseApp(image: ws_only_image)
```
Comment (jon-betts) [inline Jenkinsfile:53]: Do we maybe want to try this first without releasing it?

File: Jenkinsfile [hunk 2]
```diff
@@ -2,12 +2,14 @@
 
 @Library('pipeline-library') _
 
-def img
+def image
```
Comment (indigobravo) [inline Jenkinsfile:5]: Variable name `img` has been replaced with `image`. Sensible change.

This is used to store the `h` docker image we have always created.

File: Jenkinsfile [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
+def ws_only_image
```
Comment (indigobravo) [inline Jenkinsfile:6]: New variable definition for `ws_only_image`. Used to store a dedicated image for the `h` `websocket`

File: Jenkinsfile [hunk 4]
```diff
@@ -2,12 +2,14 @@
...
 
 node {
     stage('build') {
         checkout(scm)
-        img = buildApp(name: 'hypothesis/hypothesis')
+        image = buildApp(name: 'hypothesis/hypothesis')
+        ws_only_image = buildApp(name: 'hypothesis/hypothesis-ws', path='h/streamer')
```
Comment (indigobravo) [inline Jenkinsfile:12]: I am wondering if the way `path='h/streamer'` is being defined is correct syntax. Should it not be: `path:  'h/streamer'`?

File: Jenkinsfile (hunk 1 above)
Comment (indigobravo) [reply Jenkinsfile:53]: As we are going to be pushing a second image, we are going to need a new repo on docker to host it. Once that has been setup, I think we should be good to release it as part of the pipeline. The QA environment will get updated with the H image it has always used, and will not know about the new `ws_only_image`.

I will create us a new repo called: `hypothesis-ws`

Comment (indigobravo) [reply Jenkinsfile:53]: https://hub.docker.com/repository/docker/hypothesis/hypothesis-ws/

**Review by indigobravo — COMMENTED** [review]
Other than the failing tests, which I suspect are related to https://github.com/hypothesis/pipeline-library/pull/6. All is looking good. I had one syntax query regarding line 12.

```
ws_only_image = buildApp(name: 'hypothesis/hypothesis-ws', path='h/streamer')
```

I am wondering if the way `path='h/streamer'` is being defined is correct syntax. Should it not be: `path: 'h/streamer'`?

File: Jenkinsfile (hunk 4 above)
Comment (jon-betts) [reply Jenkinsfile:12]: Yah... had my Python head on...

**Review by indigobravo — APPROVED** [review]
I have taken a look at the basics and the change seems sensible. I notice the makefile will need to be updated before this is merged. Interested to hear Rob's feedback.

**Review by robertknight — APPROVED** [review]

**Pull request merged**